ui-components/src/page_components/listing/listing_sidebar/Apply.tsx

```diff
@@ -93,23 +94,32 @@ const Apply = (props: ApplyProps) => {
         )}
         {!openDateState(listing) && onlineApplicationUrl !== "" && (
           <>
-            <LinkButton
-              styleType={AppearanceStyleType.primary}
-              className="w-full mb-2"
-              href={onlineApplicationUrl}
-            >
-              {t("listings.apply.applyOnline")}
-            </LinkButton>
+            {preview ? (
+              <Button disabled className="w-full mb-2">
+                {t("listings.apply.applyOnline")}
+              </Button>
+            ) : (
+              <LinkButton
```

Comment:
Do you think there's value in adding a disabled prop to the LinkButton component instead of toggling here?

Reply:
That was my initial impulse, but LinkButton is just an anchor tag in the end which doesn't have a "disabled" attribute in terms of the HTML. We could possibly figure out a workaround, but I'm not sure if having a "disabled" anchor tag is all that common of a use case…